src/app_builder.rs

```diff
@@ -10,8 +10,10 @@ use schemars::JsonSchema;
 use serde::de::DeserializeOwned;
 use std::fmt::Debug;
 
-/// This is essential to create a custom app with custom handler.
-///   let mut app = BasicAppBuilder::<E, Q>::new_custom().with_custom(handler).build();
+/// This is essential to create a custom app with custom handler, like this:
+/// ```text
```

Comment:
\`\`\`rust

Reply:
Added Rust example of instantiating BasicAppBuilder.